src/app/containers/MostWatched/index.test.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { arrayOf, shape, bool } from 'prop-types';
+import { render } from '@testing-library/react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+import { RequestContextProvider } from '#contexts/RequestContext';
+import { storyItem } from '#models/propTypes/storyItem';
+import MostWatched from '.';
+import mostWatchedData from '#data/pidgin/mostWatched/index.json';
+
+const promos = mostWatchedData.records.slice(0, 5).map(item => item.promo);
+const MostWatchedComponent = ({ data, isAmp }) => (
+  <ServiceContextProvider service="pidgin">
+    <RequestContextProvider
+      bbcOrigin="https://www.test.bbc.co.uk"
+      isAmp={isAmp}
+      pageType="MAP"
+      pathname="/pidgin/tori-49450859"
+      service="pidgin"
+      statusCode={200}
+    >
+      <MostWatched data={data} />
+    </RequestContextProvider>
+  </ServiceContextProvider>
+);
+
+MostWatchedComponent.propTypes = {
+  data: arrayOf(shape(storyItem)).isRequired,
+  isAmp: bool.isRequired,
+};
+
+describe('MostWatched', () => {
+  shouldMatchSnapshot(
```

Comment:
Could we have a non snapshot test for this? I increasingly think snapshots, unless they are focused on small bits of the UI, are just noise and a pain to constantly have to update.